include/SH3/arc/vfile.hpp

```diff
@@ -50,6 +50,7 @@ namespace sh3 { namespace arc {
         };
 
 
+        vfile() : fpos(0), fname(""){}
```

Comment:
Integers are initialized to 0 by default. `std::string` is initialized to the empty string by default as well.